test/middleware.spec.ts

```diff
@@ -43,6 +30,18 @@ describe("middleware", () => {
     mode: "active",
     type: "message",
   };
+  const webhookSignature = {
+    "X-Line-Signature": "GzU7H3qOXDzDD6cNcS/9otLzlLFxnYYriz62rNu5BDE=",
```

Comment:
shouldn't it be `x-line-signature` no?

Reply:
It's sent by server and http headers are case-insensitive.

Reply:
LGTM